dymos/examples/cannonball/doc/test_doc_cannonball_implicit_duration.py

```diff
@@ -155,7 +140,8 @@ def compute(self, inputs, outputs):
 
         traj = p.model.add_subsystem('traj', dm.Trajectory())
 
-        transcription = dm.Radau(num_segments=20, order=3, compressed=False)
+        # transcription = dm.Radau(num_segments=20, order=3, compressed=False)
+        transcription = dm.PicardShooting(num_segments=5, order=3, compressed=False)
```

Comment:
Maybe add a separate test for Picard, unless this is redundant

Reply:
In this particular one I think we just want picard. It relies on the trajectory being physically valid at each iteration.  Will removed the commented line.